package-lock.json

```diff
@@ -6617,17 +6618,18 @@
       "devDependencies": {
         "@apollo/client": "3.8.1",
         "@eclipse-sirius/sirius-components-core": "*",
+        "@eclipse-sirius/sirius-components-selection": "*",
```

Comment:
Are you sur you need the dependency on components-selection for deck ?
I see gantt have this dependency too, but it's not related to the explorer selection, it's a representation use to display a modal.